ethanschreur/kids-krafts
# Sellers can delete purchases

**What does this PR do?**
- This pull request allows sellers to delete customer purchases

**Description of the task to be completed** 
- Add a delete button to the edit purchases forms
- Make that delete button cause the purchase to be deleted
- Pressing the delete button should reload the order page

**How should this  be tested**
- (On Windows) Run `. venv/scripts/activate` 
- `python run.py`
- Go to http://127.0.0.1:5000/orders (log in if you have to)
- Fill out the "Add Orders" form and enter it
- Press on that order in the orders table
- Fill out the "Add Purchase" form and enter it
- Delete that purchase in the "Edit Purchases" section
- Confirm that it was deleted.
